front/playwright.config.ts

```diff
@@ -37,12 +39,16 @@ export default defineConfig({
       },
       dependencies: ['setup'],
     },
-    {
-      name: 'firefox',
-      use: {
-        ...devices['Desktop Firefox'],
-      },
-      dependencies: ['setup'],
-    },
+    ...(CI_ENV === 'staging'
```

Comment:
Do we want to disable Firefox when running tests locally on a dev's machine? This can make it difficult to debug issues found when the PR is added to the merge queue.